### ISSUE TITLE
[TEST-FAILURE] ClientServiceTest Timeout

### ISSUE THREAD

**Issue by hasancelik** [hazelcast/hazelcast#6580]
```
00:33:05 Running com.hazelcast.client.ClientServiceTest
02:28:31 Build timed out (after 180 minutes). Marking the build as aborted.
```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x/3956/console

**Comment by hasancelik** [comment]
```
05:45:07 Running com.hazelcast.client.ClientServiceTest
07:14:49 Build timed out (after 180 minutes). Marking the build as aborted.
```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-OracleJDK8/606/console

**Comment by hasancelik** [comment]
After the #6635 , these test failed at Jenkins @cangencer 

com.hazelcast.client.ClientServiceTest.testClientListenerDisconnected
com.hazelcast.client.ClientServiceTest.testClientListenerDisconnected
com.hazelcast.client.ClientServiceTest.com.hazelcast.client.ClientServiceTest

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-OracleJDK1.7/293/

**Comment by hasancelik** [comment]
```
java.lang.AssertionError: CountDownLatch failed to complete within 30 seconds , count left: 2
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:746)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:739)
	at com.hazelcast.client.ClientServiceTest.testClientListenerDisconnected(ClientServiceTest.java:335)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-OracleJDK1.6/com.hazelcast$hazelcast-client/719/testReport/junit/com.hazelcast.client/ClientServiceTest/testClientListenerDisconnected/

**Comment by hasancelik** [comment]
#6694 
#6695
These two tests related this failure.

**Comment by sancar** [comment]
fixed by #6728
